packages/website/src/components/ast/utils.ts

```diff
@@ -227,8 +228,12 @@ export function filterProperties(
   }
 
   switch (type) {
-    case 'esNode':
-      return key !== 'tokens' && key !== 'comments';
+    case 'esNode': {
+      if (key === 'tokens') {
+        return showTokens === true;
+      }
+      return true;
```

Comment:
[Nitpick] Could simplify this a teeny bit, if you're interested?

```suggestion
      return key !== 'tokens' || !!showTokens;
```